client/src/containers/projects/table/view/key-costs/columns.tsx

```diff
@@ -4,12 +4,12 @@ import { createColumnHelper } from "@tanstack/react-table";
 const columnHelper = createColumnHelper<
   Partial<Project> & {
     // ! these types should be part of the Project entity eventually, we are adding them here to silent TS for now
-    implementationLabor: number;
-    communityBenefitSharingFund: number;
-    monitoringAndMaintenance: number;
-    communityRepresentationLiaison: number;
-    conservationPlanningAndAdmin: number;
-    carbonStandardFees: number;
+    // implementationLabor: number;
```

Comment:
These properties do not exist as part of the Project entity so it break the transpilation, this is legit